go/client/cas.go

```diff
@@ -39,14 +41,14 @@ func (c *Client) WriteBlobs(ctx context.Context, blobs map[digest.Digest][]byte)
 	if err != nil {
 		return err
 	}
-	log.V(1).Infof("%d blobs to store", len(missing))
+	log.V(2).Infof("%d blobs to store", len(missing))
 	var batches [][]digest.Digest
 	if c.useBatchOps {
 		batches = makeBatches(missing)
 	} else {
-		log.V(1).Info("uploading them individually")
```

Comment:
Logs should start with a capital letter here and elsewhere

Reply:
Done.